src/static/js/modules/util/utilities-type-checkers.js

```diff
@@ -0,0 +1,150 @@
+/* ==========================================================================
+   Javascript Type Checkers
+
+   Various utility functions to check Javascript types and primitives.
+   Copied from:
+   https://github.com/angular/angular.js/blob/master/src/Angular.js.
```

Comment:
Yeah, I wasn't sure because I've modified the original for ESLint purposes. I'll add it in the comment section.